libs/backend/src/system_call/export_logs_to_usb.ts

```diff
@@ -249,11 +250,31 @@ export async function exportLogsToUsb({
     logger,
   });
 
+  if (result.isErr()) {
+    const error = result.err();
+
+    let cause: string | undefined;
+    if ('cause' in error) {
+      if (error.cause instanceof Error) {
+        cause = error.cause.stack || error.cause.toString();
+      } else {
+        cause = `${error.cause}`;
+      }
+    }
```

Comment:
Consider using `extractErrorMessage` from libs/basics

Reply:
Oh nice, didn't know that existed - thanks! Looks like it prioritizes the message over the stack trace, while I want the reverse here, but can use it for the fallbacks